lib/processor.spec.js

```diff
@@ -0,0 +1,203 @@
+import test from 'ava'
```

Comment:
Just out of curiosity: Where is this spec on the same level as the file coming from? Never seen that before in js-land.